e2e_playwright/st_checkbox_test.py

```diff
@@ -52,14 +52,17 @@ def test_checkbox_initial_values(app: Page):
 
 def test_checkbox_values_on_click(app: Page):
     """Test that st.checkbox updates values correctly when user clicks."""
-    checkbox_elements = app.locator(".stCheckbox")
+    checkbox_elements = app.get_by_test_id("stCheckbox")
     expect(checkbox_elements).to_have_count(8)
 
     for checkbox_element in checkbox_elements.all():
+        # Not sure if this is needed, but somehow it is slightly
+        # flaky with the last toggle without it.
```

Comment:
Is there any more information that we can add here? I read this and am wondering if we read this in the future and there's enough context. Is there a link to a github run maybe?

Reply:
See the other comment on the toggle test.